src/partials/footer.html

```diff
@@ -1,29 +1,30 @@
 <footer>
-    <div>
-    <h2>HOMEMADE ICE CREAM MADE<span> WITH PASSION</span></h2>
-      <ul>
-        <li>
-          <a href="https:/github.com/" target="_blank"
-            ><svg><use></use></svg></a
-          >
-        </li>
-        <li>
-          <a href="https://telegram.org/" target="_blank"
-            ><svg><use>tel</use></svg></a
-          >
-        </li>
-        <li>
-          <a href="https://facebook.com/" target="_blank"
-            ><svg><use>fb</use></svg></a
-          >
-        </li>
-        <li>
-          <a  href="tel:+610383766284"
-            >Feel free to contact <span>+61 (0) 383 766 284</span></a
-          >
-        </li>
-      </ul>
-    </div>
-    <span>&copy; 2019 - IceCream. All Rights Reserved.</span>
+  <div class="footer">
+    <h2 class="footer__header">
+      HOMEMADE ICE CREAM MADE<span class="footer__header--color"> WITH PASSION</span>
+    </h2>
+    <ul class="footer__list">
+      <li class="footer__link">
+        <a class="footer__circle" href="https:/github.com/" target="_blank"
+          ><svg width="32" height="32"><use href="./images/icons.svg#icon-github"></use></svg>
+        </a>
+      </li>
+      <li class="footer__link">
+        <a class="footer__circle" href="https://telegram.org/" target="_blank"
+          ><svg width="32" height="32"><use href="./images/icons.svg#icon-telegram"></use></svg>
+        </a>
+      </li>
+      <li class="footer__link">
+        <a class="footer__circle" href="https://facebook.com/" target="_blank"
+          ><svg width="32" height="32"><use href="./images/icons.svg#icon-facebook"></use></svg>
+        </a>
+      </li>
+    </ul>
+    <span>
+      <a class="footer__contacts" href="tel:+610383766284"
+        >Feel free to contact <span class="footer__contacts--color"> + 61 (0) 383 766 284</span></a
```

Comment:
modyfikator nie może występować bez elementu modyfikowanego

Reply:
to już pisałeś wyżej